test/cljam/io/bam/encoder_test.clj

```diff
@@ -112,3 +114,105 @@
        0x00 0x00 0x00 0x00
        0x00 0x00 0xb8 0x40
        0xff 0xff 0x7f 0x7f])))
+
+(deftest add-cigar-to-options-test
+  (are [cigar expected-option-list sample-option-list]
```

Comment:
If you put the expected value in the middle, we would hardly tell which value is which at a glance, so I think

```
[cigar sample-option-list expected-option-list]
```
or
```
[expected-option-list cigar sample-option-list]
```
would be preferable.

Also, to be consistent with other test cases, these placeholder names should be prefixed with `?`.